### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by billtlee** [AndyOsei/react-native-meetio#3]
Hi Andy,

This looks great!  Would it be possible to add tab navigation for iOS and drawer navigation for Android in future releases?

**Comment by AndyOsei** [comment]
yes, there are actually both drawer & tab experiences in the UI kit, so will build them when I get to those portions.
